modules/op-starting-pipelines.adoc

```diff
@@ -38,17 +38,17 @@ Select *Add Credentials* if you want to specify an additional *Registry Server A
 +
 You can add multiple secrets based upon the number of resources in your Pipeline.
 
-. Click *Start* to start the PipelineRun.
-. The *Pipeline Run Details* page displays the Pipeline being executed. After the Pipeline starts, the Tasks and Steps within each Task are executed.
-You can:
-
-* Hover over the Tasks to see the time taken for the execution of each Step.
-* Click on a Task to see logs for each of the Steps in the Task.
-* Click the *Logs* tab to see the logs according to the execution sequence of the Tasks and use the *Download* button to download the logs to a text file.
+. Click *Start* to start the pipeline run.
+. The *Pipeline Run Details* page displays the pipeline being executed. After the pipeline starts, the tasks and steps within each task are executed. You can:
+* Hover over the tasks to see the time taken for the execution of each step.
+* Click on a task to see logs for each of the steps in the task.
+* Click the *Task Runs* tab to see all the task runs in a pipeline. To see the details of a task run, click on the name of the task run.
+* Click the *Logs* tab to see the logs according to the execution sequence of the Tasks, and use the *Download* button to download the logs to a text file.
```

Comment:
There are three button on the top right corner in the logs page.
![image](https://user-images.githubusercontent.com/9964343/109951483-bfbadb00-7d03-11eb-8149-d5b327541fb8.png)
1. `Download` will download the currently running task.
2. `Download All` will download the all executed task logs
3. `Expand/Collapse` button will be enter/exit the full screen mode.

Reply:
Explaining/mentioning all buttons will be too much dumbing down - I have included a generic statement.